seccomp/chown.go

```diff
@@ -0,0 +1,205 @@
+//
+// Copyright 2019-2020 Nestybox, Inc.
```

Comment:
minor: new files should just expose current year.

Reply:
Done.